roles/rke/defaults/main.yml

```diff
@@ -4,7 +4,7 @@
 kube_cloud_provider: "{{ lookup('env', 'KUBE_CLOUD_PROVIDER') }}"
 kube_cloud_conf: "{{ lookup('env', 'KUBE_CLOUD_CONF') | b64decode }}"
 # Whether to use the in-tree cloud provider or external cloud-provider
-kube_in_tree_provider: true
+kube_in_tree_provider: false
```

Comment:
Won't this break the other clouds?  Did you check the csi drivers as well?

Reply:
CSI drivers work for openstack JS2. Haven't looked at other clouds, but I also imagine they're a lot less needed given they have k8s services, so at least personally I think I'm only using this stack for JS